### PR TITLE
ganerate custom test report based on env.

### PR DESCRIPTION
**Pull request by morvencao**
generate custom test report based on `JUNIT_REPORT_FILE` environment variable, normally we use test report from ginkgo, but for e2e test on aro and rosa, we need custom test report compatible with QE report portal.

### CODE REVIEW

File: test/e2e/pkg/suite_test.go [hunk 1]
```diff
@@ -175,6 +176,16 @@ var _ = AfterSuite(func() {
 	cancel()
 })
 
+var _ = ReportAfterSuite("GlobalHub e2e Test Report", func(report Report) {
```
Comment (hchenxa) [inline test/e2e/pkg/suite_test.go:179]: change the globalhub to Maestro

Comment (morvencao) [reply test/e2e/pkg/suite_test.go]: done.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**